src/util/generator.ts

```diff
@@ -315,13 +315,15 @@ export const createZip = async (
   zip.file('test/jest.config.js', jestConfig);
   zip.file('test/index.test.js', testFile);
 
-  const { accounts, transactionTemplates, scriptTemplates } = project;
+  const { contracts, transactionTemplates, scriptTemplates } = project;
 
-  for (let i = 0; i < accounts.length; i++) {
-    const account = accounts[i];
-    const name = getContractName(account.draftCode);
-    const fileName = `cadence/contracts/${name}.cdc`;
-    zip.file(fileName, account.draftCode);
+  // loop through multiple contracts per account
+  // and put them in folders
+  for (let i = 0; i < contracts.length; i++) {
+    const contract = contracts[i];
+    const name = getContractName(contract.script);
+    const fileName = `cadence/contracts/0x0${contract.index+1}/${name}.cdc`;
```

Comment:
I would probably make this one a bit more elaborate and only add prefix/suffix if there is name collision, but this will do for now :)

Reply:
yes, noted. a follow up question, is it possible to import the exported project folder somewhere?